packages/office-ui-fabric-react/src/components/ComboBox/ComboBox.styles.ts

```diff
@@ -1,321 +1,316 @@
-import {

```

Comment:
Was this file added by accident? Looks like it might have gotten changed by some autoformatting?

Reply:
Oh man, yes, good catch. I'll see what I can do to clean that up!

Reply:
Looks like it came through in a merge on this commit:
ce1e22a514fbc7aff30089d3a2c2bd5acb7c052c

Reply:
hmmm...but why when merging into master does it show as a change?

Reply:
Very good question! Trying to compare more closely...

Reply:
Ooook, after backing up, I remembered that I had resolved a conflict with ComboBox.styles, and somewhere in that process the diff overreacted. I re-resolved the conflict, and the diff is looking cleaner. The change is per the 3rd comment above (Style changes also include removing error '*' in ComboBox, to address this issue: #2894). Here is an extra screen shot of the new "error message": 
![image](https://user-images.githubusercontent.com/16619843/30924423-88c6adb8-a363-11e7-97bd-7409b83b54a1.png)